tests/README.md

```diff
@@ -31,6 +31,26 @@ TC_PATHS="tests/kafkatest/tests/streams tests/kafkatest/tests/tools" bash tests/
   - The docker containers are named knode01, knode02 etc.
    These nodes can't be used for any other purpose.
 
+Examining CI run
+----------------
+* Set BUILD_ID is travis ci's build id. E.g. build id is 169519874 for the following build
+```
+https://travis-ci.org/apache/kafka/builds/169519874
+```
+
+* Getting number of tests that were actually run
+```
+for id in $(curl -sSL https://api.travis-ci.org/builds/$BUILD_ID | jq '.matrix|map(.id)|.[]'); do curl -sSL "https://api.travis-ci.org/jobs/$id/log.txt?deansi=true" ; done | egrep 'RunnerClient: Loading test' | wc
```

Comment:
@ewencp Would it be a problem if I use urls like:
https://travis-ci.org/raghavgautam/kafka/builds/191714520
This way I will be able to put working examples in the documentation.
Note: raghavgautam in the url